README.md

```diff
@@ -59,6 +59,70 @@ f1.content = "The quick brown fox jumped over the lazy dog."
 bf1.save
 ```
 
+## Extending Directly Contained Derivatives
+
+Hydra Works comes with the ability to attach [Hydra Derivatives](https://github.com/projecthydra/hydra-derivatives) to a GenericFile. This allows a GenericFile of an image, video, audio file, or document type to directly contain appropriate derivative files such as thumbnail, extracted_text, mp3, ogg, mp4, and webm. The derivatives are configured in `lib/hydra/works/models/concerns/generic_file/contained_files.rb` and `lib/hydra/works/models/concerns/generic_file/derivatives.rb`.
+
+More directly contained derivatives can be configured for the Persist Derivatives service by extending `contained_files` and `makes_derivatives`, as long as the processor is defined in Hydra Derivatives.
+
+### contained_files
+```
+# lib/hydra/works/models/concerns/generic_file/contained_files.rb
```

Comment:
@terrellt raises a good point about inheritance, and I imagine that folks will want to tweak this on a model-by-model basis rather than say "I want markdown derivs for everything."  I prefer the pattern he suggests to the new one I suggested, and it has the advantages my pattern does along with symmetry with all the other code in Hydraland that deals with derivatives (e.g., https://github.com/psu-stewardship/archivesphere/blob/develop/app/models/concerns/archival_derivatives.rb).